examples/cookbooks/secrets_management/README.md

```diff
@@ -0,0 +1,45 @@
+# HowTo: Retrieving secrets from a secrets management tool
+This cookbook will store examples on how to grab secrets from tools like Hashicorp Vault. In the future I would like to include examples for Akeyless.io/Aws KMS/Azure
+
+## Before you start
+### Assumptions
+* This guide assumes that you have a working Vault solution up and running that you can interact with
+
+**Note:** The way that the vault token is used in this cookbook is not the way we recommended to store the vault_token. 
```

Comment:
Let's expand upon this:

- Why do we not recommend what we're doing?
- What about what's in this cookbook is bad? 
- What sort of thing would we do here? Is there anything we'd recommend? This is a non-trivial problem to solve securely, and I'd love to see more around that.